### DISCUSSION TITLE
Where do I find the icons already being used to represent each object?

### DISCUSSION THREAD

**Discussion by dg123123** [deejay-hub/timeline-lwc#200] [Q&A] [answered]
I absolutely LOVE Timeline...for the record!

The struggle that I'm having is where I want the data point in Timeline to be represented with the same icon and color as the way each object is represented in the Related List Quick Links.

See image below:

<img width="1150" alt="2022-06-20_21-33-37" src="https://user-images.githubusercontent.com/107086809/174698560-c9a3c86a-debc-4957-9d08-e1a2f67514cd.png">

I can't find these icons anywhere.

Where do I find the corresponding URLs for any of the icons that were selected for any of these objects?

Thanks in advance!

**Comment by deejay-hub** [comment, marked as answer]
Good question. It's a bit of a dark art. The config guide (https://quip.com/ahFCA9VBEKtr) has a section called Icon Library that has the instructions. 

An alternative approach to the docs is to use the browser developer tools (which is usually quicker) - I'll use Chrome below but something similar will work in any browser. 

Find the related list with the icon you want to replicate (Contracts) in this instance. Then right click on the icon and select Inspect.
<img width="530" alt="image" src="https://user-images.githubusercontent.com/7021883/174700311-9a70b6f8-c4f5-41d3-a7b2-2f921b1cd399.png">

Then in the window that opens you'll see two things. The url to the icon (you don't need the full url just everything from /img onwards. The second is the hex value for the colour which will be positioned above it.

<img width="1281" alt="image" src="https://user-images.githubusercontent.com/7021883/174700606-cb62fcfd-08e9-4e4d-adb8-4abfcd0f2096.png">

Having a setup page that makes this easier is on the list of things to do - we don't think this is ideal. In the meantime it's manual.

So for geo areas:
Icon: /img/icon/t4v35/custom/custom68_120.png     Colour: #f26979

So for contracts:
Icon: /img/icon/t4v35/standard/contract_120.png     Colour: #6EC06E

For remuneration
Icon: /img/icon/t4v35/custom/custom41_120.png   Colour: #43b5b5

For training
Icon: /img/icon/t4v35/custom/custom82_120.png.   Colour: #f26979

**Reply by dg123123** [reply]
@deejay-hub You're a rockstar! Thank you!
